src/lib3dc/Makefile

```diff
@@ -234,13 +234,13 @@ distuninstallcheck_listfiles = find . -type f -print
 am__distuninstallcheck_listfiles = $(distuninstallcheck_listfiles) \
   | sed 's|^\./|$(prefix)/|' | grep -v '$(infodir)/dir$$'
 distcleancheck_listfiles = find . -type f -print
-ACLOCAL = ${SHELL} '/home/bbaker/git/github/spacepilotpro-lcd/src/lib3dc/config/missing' aclocal-1.16
+ACLOCAL = ${SHELL} '/home/joe/git/spacepilotpro-lcd/src/lib3dc/config/missing' aclocal-1.16
```

Comment:
Ive never seen this before, im not even sure what im looking at, why does the makefile change?